account_invoice_refund_line_selection/tests/test_refund_line.py

```diff
@@ -0,0 +1,138 @@
+# Copyright 2019 Creu Blanca
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from odoo.tests import tagged
+
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+
+
+@tagged("post_install", "-at_install")
+class TestInvoiceRefundLine(AccountTestInvoicingCommon):
+    @classmethod
+    def setUpClass(cls, chart_template_ref=None):
+        super().setUpClass()
+        cls.env = cls.env(
```

Comment:
I think you don't need to mute mail-related context, because it's already done in `BaseCommon`
See: `class AccountTestInvoicingCommon(ProductCommon)` -> `class ProductCommon(BaseCommon, UomCommon)`